tpcx_bb/xbb_tools/utils.py

```diff
@@ -772,15 +772,15 @@ def build_benchmark_googlesheet_payload(config):
     payload = OrderedDict(
         {
             "Query Number": QUERY_NUM,
-            "Protocol": data.get("protocol"),
+            "Protocol": "UCX" if data.get("nvlink") == True else "TCP",
```

Comment:
Should this be if it's either IB or NVLINK, or is the assumption here that we don't use IB without NVLINK (Which is true but maybe add a comment to clear this up ?)

Reply:
Assumption is that if NVLink is enabled we must be using UCX. We may be using IB, but may not be. We have another dictionary key to capture IB, though it's not doing anything at the moment.

Reply:
Gotcha, That makes sense.